### DISCUSSION TITLE
Welcome to community Discussions!

### DISCUSSION THREAD

**Discussion by redhatrises** [CrowdStrike/community#26] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by amal2041998** [comment]
Google Workspace has been successfully integrated with CrowdStrike NG-SIEM, and I’m looking to implement a relevant dashboard. If anyone has a custom dashboard configuration or template, please share it
